pkg/scheduler/plugins/proportion/proportion.go

```diff
@@ -242,7 +242,7 @@ func (pp *proportionPlugin) OnSessionOpen(ssn *framework.Session) {
 		queue := obj.(*api.QueueInfo)
 		attr := pp.queueOpts[queue.UID]
 
-		overused := !attr.allocated.LessEqual(attr.deserved)
+		overused := !attr.allocated.Less(attr.deserved)
```

Comment:
Please make CI happy

Reply:
#1036 lead the CI failed, so need to fix it first.